packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -392,6 +393,90 @@ export const MEDIA_UPDATE_TYPE = {
  * @class Meeting
  */
 export default class Meeting extends StatelessWebexPlugin {
+  attrs: any;
+  audio: any;
+  conversationUrl: string;
+  correlationId: string;
+  destination: string;
+  destinationType: string;
+  deviceUrl: string;
+  effects: any;
+  hostId: string;
+  id: string;
+  locusUrl: string;
+  mediaConnections: any[];
+  meetingFiniteStateMachine: any;
+  meetingInfo: object;
+  meetingRequest: any;
+  members: Members;
+  options: object;
+  orgId: string;
+  owner: string;
+  partner: any;
+  policy: string;
+  reconnectionManager: ReconnectionManager;
+  resource: string;
+  roap: Roap;
+  roapSeq: number;
+  sipUri: string;
+  type: string;
+  userId: string;
+  video: any;
+  callEvents: any[];
+  deferJoin: Promise<any>;
+  dialInDeviceStatus: string;
+  dialInUrl: string;
+  dialOutDeviceStatus: string;
+  dialOutUrl: string;
+  fetchMeetingInfoTimeoutId: NodeJS.Timeout;
+  floorGrantPending: boolean;
+  hasJoinedOnce: boolean;
+  hasWebsocketConnected: boolean;
+  inMeetingActions: InMeetingActions;
+  isLocalShareLive: boolean;
+  isSharing: boolean;
+  keepAliveTimerId: NodeJS.Timeout;
+  lastVideoLayoutInfo: any;
+  locusInfo: any;
+  mediaProperties: MediaProperties;
+  meetingInfoFailureReason: string;
+  networkQualityMonitor: NetworkQualityMonitor;
+  networkStatus: string;
+  passwordStatus: string;
+  queuedMediaUpdates: any[];
+  recording: any;
+  requiredCaptcha: any;
+  shareStatus: string;
+  statsAnalyzer: StatsAnalyzer;
+  transcription: Transcription;
+  updateMediaConnections: (mediaConnections: any[]) => void;
+  endCallInitiateJoinReq: any;
+  endJoinReqResp: any;
+  endLocalSDPGenRemoteSDPRecvDelay: any;
+  joinedWith: any;
+  locusId: any;
+  startCallInitiateJoinReq: any;
+  startJoinReqResp: any;
+  startLocalSDPGenRemoteSDPRecvDelay: any;
+  wirelessShare: any;
+  guest: any;
+  meetingJoinUrl: any;
+  meetingNumber: any;
+  meetingState: any;
+  permissionToken: any;
+  resourceId: any;
+  resourceUrl: string;
+  selfId: string;
+  state: any;  
+
+  // JSTOTS: VERIFY DECLARED VARIABLES WHETHER ALL OF THEM BELONG HERE OR EXTENDED FROM PARENT CLASS  
+  // currentMediaStatus: any;
```

Comment:
why do we have these commented out properties here?

Reply:
Fixed.